ConsoleApp/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿static bool Compare(string first, string second)
+{
+    if (first.Length != second.Length)
+    {
+        return false;
+    }
+    else if (first.Length == second.Length)
+    {
+        for (int i = 0; i < first.Length; i++)
+        {
+            if (first[i] != second[i])
+            {
+                return false;
+            }
+            else if (first[i] == second[i])
+            {
+                continue;
+            }
+        }
+
+    }
+    return true;
+}
+
+static void Analyze(string first)
+{
+    int charcount = 0, digitcount = 0, symbolcount = 0;
+
+    foreach (var item in first)
+    {
+        if (Char.IsLetter(item))
+        {
+            charcount++;
+        }
+        else if (Char.IsDigit(item))
+        {
+            digitcount++;
+        }
+        else if (Char.IsSymbol(item) || Char.IsPunctuation(item))
+        {
+            symbolcount++;
+        }
+    }
+
+    Console.WriteLine($"Букв у масиві : {charcount}");
+    Console.WriteLine($"Цифр у масиві : {digitcount}");
+    Console.WriteLine($"Символів у масиві : {symbolcount}");
+}
+
+static string Sort(string first)
+
+{
+    string sorted = first.ToLower();
+
+    char[] temparray = new char[sorted.Length];
```

Comment:
you can use here next:
```suggestion
    char[] temparray = first.ToLower().ToCharArray();
```